pontoon/contributors/templates/contributors/settings.html

```diff
@@ -41,7 +41,6 @@ <h2 id="username">{{ user.first_name }}</h2>
 {% block bottom %}
 <section id="main">
   <div id="profile-form">
-      {% csrf_token %}
```

Comment:
Pretty sure that is not the right answer to the problem. Removing the csrf token is going to expose us to attacks, and django doesn't even allow it. 

Sadly, it seems our JS code allows user data editing without requiring the CSRF token. @mathjazz how come that's possible? Isn't that a security issue?

Reply:
Not sure exactly, but I can tell you how to reproduce: 

1. checkout this branch
2. go to your settings page
3. change your email address, and press Enter
4. the change happen, even though no CSRF token is on the page

I am absolutely not sure if this is a problem or not, which is why I'm bringing this up. I assume you've worked on this, so maybe you know?

Reply:
Token is passed here:
https://github.com/mozilla/pontoon/pull/631/files#diff-10c87556f33a2cada9ff50520978ed6dL11

Reply:
Aha, OK, the CSRF token is added as a data argument in another HTML tag, perfect. That explains this, and there's no security issue. Thanks, @mathjazz !